Documentation/building/windows-instructions.md

```diff
@@ -76,6 +76,23 @@ For more details, or to test an individual project, see the [developer guide top
 4. Set breakpoint appropriately
 5. F5 (Debug)
 
+### Debugging NETFX tests in Visual Studio
+
+Once you've build the source code for netfx from the root (`build.cmd -framework:netfx`) follow this steps:
+
+1. Build the test project through msbuild /p:targetgroup=netfx
+2. Open Visual Studio
+3. Open xunit runner for your test project `%corefxpath%\bin\\%OSGroup.Architecture.BuildConfiguration%\\%YOURPROJECT%\\%TARGETGROUP%\xunit.console.exe`
```

Comment:
Yes that would be fine. I also generally like to give a full path example so that folks get a feel for what the path looks like.